Makefile

```diff
@@ -9,7 +9,7 @@ build:
 	go build -o osbuild-upload-aws ./cmd/osbuild-upload-aws/
 	go build -o osbuild-tests ./cmd/osbuild-tests/
 	go build -o osbuild-weldr-tests ./cmd/osbuild-weldr-tests/
-	go build -o osbuild-dnf-json-tests ./cmd/osbuild-dnf-json-tests/
+	go test -c -o osbuild-dnf-json-tests ./cmd/osbuild-dnf-json-tests/main_test.go
```

Comment:
Does this work without the tag? What's the purpose of this make target?

Reply:
It does not. I don't usually use this target, that's why I missed this.

I've fixed it to include the tag here. Whether to remove the `make` target is a separate discussion, I'd say.

Reply:
@teg it looks like some folks are using this target when working on new test binaries, myself included.